tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -50,6 +50,12 @@ const (
 	vnetRuleName = "vnet_traffic"
 	// winUser is the user used to login into the instance.
 	winUser = "core"
+	// sshPort is the SSH port
+	sshPort = string(types.SshPort)
```

Comment:
Why this explicit type casting? Ideally you should have type casting within the function by creating another var